### DISCUSSION TITLE
Can meshing happen between two different networks

### DISCUSSION THREAD

**Discussion by ksathara** [openthread/openthread#10349] [Q&A]
Hi, 
Suppose I have two different OpenThread networks (either different PAN IDs or network keys), can we have meshing between them? 
This means, can one network's data packet route via the other? This is just to enlarge the range.

Cheers,
Kaushalya

**Comment by jwhui** [comment]
A Thread network interface can only participate in a single Thread network at a time. A device can incorporate multiple Thread network interfaces and establish routes between the two Thread networks. Alternatively, two border routers (each serving a different Thread network) can be connected over an infrastructure link (e.g. Wi-Fi or Ethernet) and route packets to each other.

**Reply by ksathara** [reply]
Hi Jonathan, Thanks. At the moment I dont have and border routers, just a couple of FTDs and couple of SEDs. Lets say I have FTD1, SED11 and SED12 on one network and FTD2, SED21 and SED22 on another network. Can SED22 route via FTD1 to reach FTD2? 
How can FTD1 and FTD2 have multiple thread network interfaces? May be I read you wrong.

Cheers,
Kaushalya

**Reply by jwhui** [reply]
A Thread network interface can only participate in a single Thread network at a time. I assume your FTDs only support a single Thread interface. If so, then they cannot connect to multiple Thread networks at the same time.
